README.md

```diff
@@ -70,14 +77,18 @@ The scripts will:
    for image storage.
 1. [Deploy
    CNV](https://github.com/openshift-kni/install-scripts/blob/master/CNV/deploy-cnv.sh). [Configure
-   a bridge on the `baremetal` interface on OpenShift nodes
+   a bridge on the `External` interface on OpenShift nodes
    (#18)](https://github.com/openshift-kni/install-scripts/issues/18)
    to allow VMs access this network.
 1. Temporarily install Ripsaw, carry out some performance tests, and
    capture the results.
 
+
+
+
 The following environment-specific information will be required for
-each installation:
+each installation.  Most of these items will be discovered in a properly
```

Comment:
"Most of these items... " change to: "The following items are discovered in a properly configured cluster:"